packages/core-ethereum/src/commitment.ts

```diff
@@ -57,14 +70,45 @@ async function createCommitmentChain(db: HoprDB, channelId: Hash, setChainCommit
   log('commitment chain initialized')
 }
 
+/**
+ * Simple class encapsulating channel information
+ * used to generate the initial channel commitment.
+ */
+export class ChannelCommitmentInfo {
```

Comment:
Not sure this should be exported

Reply:
Right now it needs to be exported, because it's used here: https://github.com/hoprnet/hoprnet/blob/3bdda1942c22810f04e5a7dd49ea8a42aa22090c/packages/core-ethereum/src/index.ts#L179

Unless of course we make it just arguments to `initializeCommitment`. However, I'd prefer to keep it encapsulated in a class, just in case we need to extend the amount information. That way we preserve the # of args of `initializeCommitment`.